package.json

```diff
@@ -4,22 +4,33 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "webpack"
-  },
+    "build": "webpack",
+    "start": "webpack serve --open"
+  }
+  ,
   "repository": {
     "type": "git",
     "url": "git+https://github.com/kimothokamau/todo_list_js.git"
-  },
+  }
+  ,
   "keywords": [],
   "author": "",
   "license": "ISC",
   "bugs": {
-    "url": "https://github.com/kimothokamau/todo_list_js/issues"
-  },
+
+  }
+  ,
   "homepage": "https://github.com/kimothokamau/todo_list_js#readme",
   "devDependencies": {
-    "html-webpack-plugin": "^5.3.2",
-    "webpack": "^5.44.0",
+    "css-loader": "^5.2.7",
+    "html-webpack-plugin": "^5.3.2"
+    ,
+    "style-loader": "^3.1.0"
+    ,
+    "webpack": "^5.44.0"
+    ,
     "webpack-cli": "^4.7.2"
+    ,
+    "webpack-dev-server": "^3.11.2"
   }
 }
```

Comment:
- Kindly fix this error when i try to run your project locally.
- Following the steps that you give in your README file.
  - run ``npm i ``
  - run ``npm run build ``
![Screen Shot 2021-07-14 at 06 40 24](https://user-images.githubusercontent.com/22258274/125616246-f0b33328-1cd1-4275-b6c0-9ffe85700043.png)